src/intents.cs

```diff
@@ -8528,5 +8702,299 @@ interface INVisualCodeTypeResolutionResult {
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface INMediaItem : NSCopying, NSSecureCoding {
+
+		[Export ("initWithIdentifier:title:type:artwork:")]
+		[DesignatedInitializer]
+		IntPtr Constructor ([NullAllowed] string identifier, [NullAllowed] string title, INMediaItemType type, [NullAllowed] INImage artwork);
+
+		[NullAllowed, Export ("identifier")]
+		string Identifier { get; }
+
+		[NullAllowed, Export ("title")]
+		string Title { get; }
+
+		[Export ("type", ArgumentSemantic.Assign)]
+		INMediaItemType Type { get; }
+
+		[NullAllowed, Export ("artwork", ArgumentSemantic.Copy)]
+		INImage Artwork { get; }
+	}
+
+	[Watch (5,0), NoMac, iOS (12,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface INObject : INSpeakable, NSCopying, NSSecureCoding {
+
+		[Export ("initWithIdentifier:displayString:pronunciationHint:")]
+		[DesignatedInitializer]
+		IntPtr Constructor ([NullAllowed] string identifier, string displayString, [NullAllowed] string pronunciationHint);
+
+		[Export ("initWithIdentifier:displayString:")]
+		IntPtr Constructor ([NullAllowed] string identifier, string displayString);
+
+		// Inlined by INSpeakable
+		//[NullAllowed, Export ("identifier", ArgumentSemantic.Strong)]
+		//string Identifier { get; }
+
+		[Export ("displayString")]
+		string DisplayString { get; }
+
+		// Inlined by INSpeakable
+		//[NullAllowed, Export ("pronunciationHint", ArgumentSemantic.Strong)]
+		//string PronunciationHint { get; }
+	}
+
+	[Watch (5,0), NoMac, iOS (12,0)]
+	[BaseType (typeof (INIntent))]
+	[DisableDefaultCtor]
+	interface INPlayMediaIntent {
+
+		[Export ("initWithMediaItems:mediaContainer:playShuffled:playbackRepeatMode:resumePlayback:")]
+		[DesignatedInitializer]
+		IntPtr Constructor ([NullAllowed] INMediaItem [] mediaItems, [NullAllowed] INMediaItem mediaContainer, [NullAllowed, BindAs (typeof (bool?))] NSNumber playShuffled, INPlaybackRepeatMode playbackRepeatMode, [NullAllowed, BindAs (typeof (bool?))] NSNumber resumePlayback);
+
+		[NullAllowed, Export ("mediaItems", ArgumentSemantic.Copy)]
+		INMediaItem [] MediaItems { get; }
+
+		[NullAllowed, Export ("mediaContainer", ArgumentSemantic.Copy)]
+		INMediaItem MediaContainer { get; }
+
+		[BindAs (typeof (bool?))]
+		[NullAllowed, Export ("playShuffled", ArgumentSemantic.Copy)]
+		NSNumber PlayShuffled { get; }
+
+		[Export ("playbackRepeatMode", ArgumentSemantic.Assign)]
+		INPlaybackRepeatMode PlaybackRepeatMode { get; }
+
+		[BindAs (typeof (bool?))]
+		[NullAllowed, Export ("resumePlayback", ArgumentSemantic.Copy)]
+		NSNumber ResumePlayback { get; }
+	}
+
+	[Watch (5,0), NoMac, iOS (12,0)]
+	[Protocol]
+	interface INPlayMediaIntentHandling {
+
+		[Abstract]
+		[Export ("handlePlayMedia:completion:")]
+		void HandlePlayMedia (INPlayMediaIntent intent, Action<INPlayMediaIntentResponse> completion);
+
+		[Export ("confirmPlayMedia:completion:")]
+		void Confirm (INPlayMediaIntent intent, Action<INPlayMediaIntentResponse> completion);
```

Comment:
Yeah, it is `Confirm` because of this https://bugzilla.xamarin.com/show_bug.cgi?id=59164 and `HandlePlayMedia` cannot be just `Handle` because it will clash with our special property.